ext_localconf.php

```diff
@@ -17,6 +17,10 @@ function () {
             '<INCLUDE_TYPOSCRIPT: source="DIR:EXT:' . $extensionKey . '/Configuration/TsConfig/Page/" extensions="tsconfig">'
         );
 
+        \TYPO3\CMS\Core\Utility\ExtensionManagementUtility::addPageTsConfig(
```

Comment:
We usually prefer to include this via registerPageTSConfigFile() in TCA/Overrides/pages.php. But never mind. We will change this to our needs after full feature integration.

Reply:
No Problem, i can change.